stress-example/flows/index.yml

```diff
@@ -0,0 +1,31 @@
+!Flow
+with:
+  logserver: false
+pods:
+  crafter:
+    uses: pods/craft.yml
+    read_only: true
+  encoder:
+    uses: $ENCODER
+    parallel: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  vec_idx:
+    uses: pods/vec.yml
+    shards: $JINA_SHARDS
+    separated_workspace: true
+    polling: all
+    uses_reducing: _merge_all
```

Comment:
just to keep things up to date, `_merge_all` to be changed to `_merge_matches`

Reply:
Right, ill change it